packages/cli/src/cmds/prune/prune.ts

```diff
@@ -71,6 +79,15 @@ export default {
 
     const outputPath = `${argv.outputDir}/${basename(argv.file)}`;
     fs.writeFileSync(outputPath, JSON.stringify(appMap));
-    console.log('done');
+
+    let message = 'done';
+    if (argv.format === 'json' && !argv.fqids)
+      message = JSON.stringify(
+        { exclusions: Array.from(appMap.data.exclusions.values()) },
+        null,
+        2
+      );
+
+    console.log(message);
```

Comment:
Listing the exclusions is a great UX but it's not on by default. I'd recommend listing the exclusions in the text output or defaulting to `json`.

Reply:
These `exclusions` also won't include anything removed via `--filter` which is a little confusing.

The output will say:
```json
{
  "exclusions": []
}
```

Which looks like nothing was applied, even though it was successful.